js/components/portal-dashboard/account-owner.tsx

```diff
@@ -0,0 +1,23 @@
+import React from "react";
+
+import css from "../../../css/portal-dashboard/header.less";
+
+export class AccountOwnerContainer extends React.PureComponent {
```

Comment:
Not a big deal, but in Redux there is a pattern of having "container" components that know about Redux state.  I'm not sure if having "container" in the name might make it appear as if this is a Redux container.  Your call.
https://www.thegreatcodeadventure.com/the-react-plus-redux-container-pattern/

Reply:
I did change the name to ..Div instead of ..Container

Reply:
Component would be good to or even just AccountOwner.